### ISSUE TITLE
Handle chars correctly

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#147]
Example:

```
if (bytes[60] == '1') {
   ...
}
```

Current output:

```
if bytes[60] == 1 {
    ...
}
```

Which is completely wrong! It just strips down the `'` quotes.

I am not sure what is the correct Swift way, but I'm currently using the following:

```
ObjC: 'a'
Swift: "a".utf8.first!
```

Swift does have a `Character` class, but it's not easy to initialize from a character (it takes `UnicodeScalar` as an argument). This is a good way to quickly get the UTF8 (ASCII) code of the character.

If you do not wish to generate this code, please, keep the single quotes around - this is the same as the formatting - it's potentially dangerous.

**Comment by drkameleon** [comment]
Great catch. I'll do my best to resolve it as of the next one (2.3).

**Comment by drkameleon** [comment]
**Question:**

Would something like `'c'` to `Character("c")` work?

Ref: https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/StringsAndCharacters.html

**Comment by charlieMonroe** [comment]
That works, depending on what your intentions with it are :) ObjC treats the char as UInt8. In Swift, you may or may not want that behavior. I'll leave this up to you.

17. 12. 2015 v 11:06, Ioannis Zafeiropoulos <notifications@github.com>:

**Comment by drkameleon** [comment]
Well, I'm marking this as (somewhat) fixed as of the upcoming 2.3 - with the conversion as stated above.

However, it'll obviously require some further study... Let's see...
